master/buildbot/changes/gerritchangesource.py

```diff
@@ -300,12 +300,12 @@ def describe(self):
             project = "all projects"
 
         if self.branch is not None:
-           branch = "branch %s" % self.branch
+           branch = "branch '%s'" % self.branch
         else:
            branch = "all branches"
 
         if not self.process:
             status = "[NOT CONNECTED - check log]"
-        msg = ("GerritChangeSource watching the remote "
+        msg = ("GerritChangeSource watching project %s (%s) in remote "
```

Comment:
Won't this print the word "project" twice?  I would argue that it is bad practice to put the prefix "project " in the project variable above.  Ditto the branch variable.  Please change those variable to only include the project and branch strings with no added decoration.  The additional words and decorations should happen here in the msg line.